tools/bf_autocorres.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python2
+
```

Comment:
Uhh, correct me if I'm wrong, but I don't think this is supposed to have a shebang line.